tests/e2e/system_test/test_node_kernel_crash_ceph_fsync.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import random
+
+from time import sleep
+from ocs_ci.ocs import constants, node
+from ocs_ci.framework.testlib import E2ETest, tier1, bugzilla, polarion_id
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources import pod
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.helpers import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@bugzilla("2075068")
+@polarion_id("OCS-3947")
+class TestKernelCrash(E2ETest):
+    """
+    Tests to verify kernel crash
+    """
+
+    pvc_size = 1
+    original_dir = "/var/lib/www/html/"
+    result_dir = "mydir"
+
+    def creates_files(self, pod_obj):
+        while True:
+            for i in range(1, 125):
+                pod_obj.exec_cmd_on_pod(
```

Comment:
Please consider using the loop in the shell itself instead of executing 125 commands on the pod.

Reply:
+1